docs/index.js

```diff
@@ -14,6 +15,10 @@ clickExpanderEls.forEach(function(el, i) {
     expanderWidgets.push(new Expander(el, { click: true }));
 });
 
+clickNestedExpanderEls.forEach(function(el, i) {
+    expanderWidgets.push(new Expander(el, { click: true, ariaHostSelector: '.expander__host > .flyout__host' }));
```

Comment:
Maybe change '.expander__host > .flyout__host' to '.expander__host > input' or '.expander__host > .expander__aria-host'. Not sure `flyout__host` still belongs on this button now that it is nested.

Reply:
Gotcha. Makes sense.